_drafts/TEMPLATE-post-title.md

```diff
@@ -9,4 +9,12 @@ tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, ge
   - r
   - python
 ---
-##Replace this with the content for your blog post
+	In a group full of people, women are 75% less likely to speak up than a male group member. Because chances are their ideas will get rejected or gaslighted by their male peers. Now imagine how much lower that percentage is in a STEM field that’s highly maledominated. In a world where the demand for computer science workers is increasing day by day, then why has the representation of women in this field been declining over the past 30 years? This is where Girls Who Code (GWC) steps in. An organization that grows young female identifying students to join the coding field. Every session the group aims to close the gender ratio gap little by little. 
```

Comment:
1. maledominated -> male-dominated
2. I would rework the third sentence: "Yet, while the demand for computer science workers is increasing, the representation of women in this field has been declining for 30 years."
3. rework the last 2 sentences (they're a little repetitive as is): "GWC is an organization that introduces young female-identifying students to coding. In every session, the group aims to close the gender gap by creating a safe space for girls to code together!

Reply:
Thank you!